<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tabs</class>
 <widget class="QTabWidget" name="Tabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>752</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生录入</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #ffffff;
}

QPushButton {
	   color: #212121; 
        border-radius: 10px; 
		border: 1px groove gray;
	    background-color: #F5F7FA
}

 QPushButton:hover {
        background-color: #d9d9f3 ;
 }</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QWidget" name="people">
   <attribute name="title">
    <string>人脸录入</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>500</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="title">
         <string>图像</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="cameraShow">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>500</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>思源黑体 CN Regular</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">cameraShow {width：}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>160</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="title">
           <string>检测结果</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="img160Show">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>160</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="title">
           <string>身份添加</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>学号：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="id_Input">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>姓名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameInput">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>新宋体</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="openCamBtn">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>打开相机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="verifactionBtn">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true">检测并添加</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="people_textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'新宋体'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
